src/createinstancedialog.h

```diff
@@ -0,0 +1,138 @@
+#ifndef MODORGANIZER_CREATEINSTANCEDIALOG_INCLUDED
```

Comment:
There are `#pragma once` in other headers, I don't know what's the standard for MO2, but maybe be consistent between the various headers.